Gopkg.lock

```diff
@@ -126,9 +134,36 @@
   pruneopts = "NUT"
   revision = "fc88cf888a3fa99ecc23d1efc1a44284268457d3"
 
+[[projects]]
+  digest = "1:11c6c696067d3127ecf332b10f89394d386d9083f82baf71f40f2da31841a009"
+  name = "github.com/hashicorp/hcl"
```

Comment:
ron what is this stuff?

you have *something* in your u-root directory that _always_ screws up your dep update. remove that something.